ptrlib/arch/common/assembler.py

```diff
@@ -22,19 +23,23 @@ def assemble(code, bits=None, arch='intel', syntax='intel', entry=None,
         entry = 'ptrlib_main'
         code = b'.global ptrlib_main\nptrlib_main:\n' + code
 
-    if is_arch_intel(arch, bits):
+    if is_arch_intel(arch):
         if syntax.lower() == 'intel':
             code = b'.intel_syntax noprefix\n' + code
+        if bits is None:
+            bits = 64
```

Comment:
この辺
```suggestion
        if bits is None:
            bits = bit_by_arch_intel(arch)
            if bits == -1: bits = 64
```
みたいにアーキテクチャ名からビット数をguessする実装にしてもらえると :bow: